sylviawanjiku/Store_manager_Api
# #161364863  Create Token

#### What does this PR do?
Create tokens for authentication during Login
#### Description of Task to be completed?
Create a user model for user data
Create a login method for the user
Validate the user details during login
If user details are valid creates a token for authentication
Decodes the token and creates a decorator
#### How should this be manually tested?
This can be tested in postman.
Create a user and allow user to login.
If user details for login are correct you should get token created
If token is valid it should be decoded  and a decorator is returned
#### What are the relevant pivotal tracker stories?
#161364863
https://www.pivotaltracker.com/story/show/161364863
#### Screenshots (if appropriate)
#### Questions:
